### ISSUE TITLE
The underlying connection was closed: An unexpected error occurred on a receive

### ISSUE THREAD

**Issue by kvermeer** [NuGet/Home#5930]
We are setting up a new development server, but nuget has issues in visual studio 2012. It's not able to receive any packages. 
We get the error: "The underlying connection was closed: An unexpected error occurred on a receive"

Any idea how to resolve this?
The server has internet connectivity

![image](https://user-images.githubusercontent.com/6012560/30801377-5b1b486e-a1e3-11e7-88f1-718111353162.png)

I'm also not able to install any packages, probably due to the same reason.
In Package Manager Console, I run command: "Install-Package log4net"
Error: "The source at nuget.org [https://www.nuget.org/api/v2/] is unreachable. Falling back to NuGet Local Cache at C:\Users\...\AppData\Local\Nuget\Cache"

NuGet product used (NuGet.exe | VS UI | Package Manager Console | dotnet.exe): VS UI
NuGet version (x.x.x.xxx): 2.8.60318.667
VS version (if appropriate): VS 2012  (11.0.50727.26)
OS version (i.e. win10 v1607 (14393.321)): Windows Server 2012 R2   (Versie 6.3 (build 9600)
Worked before? If so, with which NuGet version: No
Package Source: https://www.nuget.org/api/v2/, but same when I try http://packages.nuget.org/v1/FeedService.svc/

**Comment by mishra14** [comment]
@kvermeer Thank you for filling a detailed issue.

Is it possible for you to move to a more recent version of [Visual Studio](https://www.visualstudio.com/downloads/)?

**Comment by kvermeer** [comment]
Thanks for your response.
Upgrading to another version of Visual Studio is not an option since we need to use a specific version of an sdk (esri arcobjects 10.2 sdk) which only works in vs 2010 and 2012.

That said, we have the issue resolved already. 
The problem was caused by some server settings related to Citrix Studio, which apparently blocked traphic with visual studio (installing extensions through visual studio gallery was also blocked)

We solved it by moving our VS installation to another server without citrix studio.
